pinot-query-planner/src/test/java/org/apache/pinot/query/QueryCompilationTest.java

```diff
@@ -529,13 +529,13 @@ private Object[][] provideQueriesWithExplainedLogicalPlan() {
             + "  \"rels\": [\n"
             + "    {\n"
             + "      \"id\": \"0\",\n"
-            + "      \"relOp\": \"LogicalTableScan\",\n"
+            + "      \"relOp\": \"org.apache.pinot.calcite.rel.logical.PinotLogicalTableScan\",\n"
```

Comment:
These are required because `RelJson` only converts to short names for calcite specific packages.